pkg/seqnum/seqnum.go

```diff
@@ -1,40 +0,0 @@
-// Package seqnum provides utilities to persistently store sequence number in a
```

Comment:
Any good reason as to why the contents of this file were moved into main?

Reply:
Reverted